src/_locales/cs.json

```diff
@@ -0,0 +1,114 @@
+  "lyrics-loading-message": "Načítání textu skladby...",
+  "lyrics-no-song-message": "Právě nehraje žádná skladba",
+
+  "message-pausing-after-song": "Pozastavení po této skladbě.",
+  "message-pausing-after-song-button": "Nepozastavovat po této skladbě",
+  "message-uncaught-error": "Nastala nespecifikovaná chyba v programu (GPMDP).",
+  "message-uncaught-error-button": "Klikněte zde pro nahlášení tohoto problému na GitHub",
+
+  "modal-confirmTray-title": "Jen abyste věděli",
+  "modal-confirmTray-content": "Zavřením hlavního okna programu (to oč jste se právě pokusili) neukončí program, ale pouze jej minimalizuje do systémové lišty. Pokud chcete tuto možnost změnit, můžete tak učinit v okně \"Nastavení Desktop Přehrávače\".",
+  "modal-confirmOpenPort-content": "Vypadá to, že Váš firewall blokuje službu PlaybackAPI. Můžeme přidat výjimku za Vás, ale upozorňujeme, že to může vyžadovat administrátorské oprávnění",
+  "modal-confirmUpdate-title": "Aktualizace je k dispozici",
+  "modal-confirmUpdate-content": "Hurá, vypadá to, že je zde nová aktualizace pro Google Play Music Desktop Player. Tato aktualizace dodá nálož nových funkcí, opravu chyb a další super věci. Doporučujeme aktualizovat co nejdříve to bude možné, abyste si mohli užít skvělé nové funkce a bezchybné prostředí. <br />Doufáme, že se Vám bude nová aktualizace líbit!",
+  "modal-confirmUninstall-title": "Takže takhle...",
+  "modal-confirmUninstall-content": "Právě jsme zjistili, že máte nainstalovanou starou verzi GPMDP. Pokud jste právě aplikaci aktualizovali, je toto oznámení běžné. Nyní se pokusíme odebrat starou verzi. To \"může\" vyžadovat oprávnění správce. Prosíme Vás o jejich potvrzení, ať můžeme uklidit ten nepořádek.",
+  "modal-goToURL-title": "Otevřete odkaz na Hudba Google Play, aby jste se dostali na:",
+  "modal-welcome-title": "Vítejte ve verzi",
+  "modal-welcome-content": "Doufáme, že se Vám nová verze líbí!",
+
+  "playback-label-next-track": "Další skladba",
+  "playback-label-play-pause": "Přehrát / Pozastavit",
+  "playback-label-previous-track": "Předchozí skladba",
+  "playback-label-stop": "Zastavit",
+  "playback-label-thumbs-down": "Palec dolů",
+  "playback-label-thumbs-up": "Palec nahoru",
+  "playback-label-volume-down": "Zeslabit hlasitost",
+  "playback-label-volume-up": "Zvýšit hlasitost",
+
+  "settings-option-custom-theme": "Vlastní motiv",
+  "settings-option-custom-theme-choose-color": "Vyberte hlavní barvu pro svůj motiv",
+  "settings-option-custom-theme-choose-custom": "Nebo si vyberte",
+  "settings-option-custom-theme-custom-color": "Vlastní barvu",
+  "settings-option-custom-theme-dark": "Tmavý motiv",
+  "settings-option-custom-theme-light": "Světlý motiv",
+
+  "settings-option-enable-api": "Povolit Playback API",
+  "settings-option-enable-api-json": "Povolit JSON API",
+  "settings-option-enable-api-details": "Tato možnost umožní jiným aplikacím komunikovat s GPMDP",

```

Comment:
I disagree here. This is a reference to the translated application. It's name is "Google Play Music Desktop Player". The author chose to use the GPMDP abbreviation for it.
I do not think it is appropiate to translate the application name as it will not be propagated to any other communications and materials.
On the other hand the sevice is Google Play Music is officially translated by Google as "Hudba Google Play" hence I used the translation where this service is referred.